vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBoxBase.java

```diff
@@ -408,14 +416,6 @@ public void setAutoOpen(boolean autoOpen) {
         getElement().setProperty("autoOpenDisabled", !autoOpen);
     }
 
-    @Override
-    public void setRequiredIndicatorVisible(boolean requiredIndicatorVisible) {
```

Comment:
Removing this breaks binary compatibility (super method is in an interface instead of class). Should not be back-ported as is.